.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,26 @@
+---
+name: ":books: Documentation"
+about: Report an issue related to missing, incorrect or insufficient documentation.
+title: ":books: [Documentation]: "
+labels: documentation
+assignees: tlvu
+---
+
+## Description
+
+<!-- Provide a description of the documentation issue. -->
+
+
+
+## References
+
+<!-- 
+  Additional screenshots or links to help identify the issue?
+-->
+
+
+| Information           | Value
+| --------------------- | --------------------------------------------------------------
+| Server/Platform URL   | <!-- Any applicable instance, e.g: https://pavics.ouranos.ca -->
+| Related issues/PR     | <!-- URL to any existing issues in https://github.com/Ouranosinc/pavics-sdi/issues ? -->
+| Related documentation | <!-- URL to incorrect https://ouranosinc.github.io/pavics-sdi/index.html HTML doc tag? -->
```

Comment:
Should be this link for doc instead https://pavics-sdi.readthedocs.io/.

https://ouranosinc.github.io/pavics-sdi/processes/workflows.html not up-to-date anymore.